README.md

```diff
@@ -1,3 +1,3 @@
-# netuitive.packages.linux.elasticsearch
+# CloudWisdom.packages.linux.elasticsearch
```

Comment:
Keep this as netuitive, it's part of a fully qualified name.

Reply:
changed back to netuitive